### DISCUSSION TITLE
A question for the roadmap

### DISCUSSION THREAD

**Discussion by MateuszLas421** [Mapsui/Mapsui#2165] [Q&A] [answered]
https://mapsui.com/documentation/roadmap.html

What do you mean by the statement in the RoadMap that in Mapsui 5, you are removing Xamarin.Android and Xamarin.iOS? Are you specifically talking about Xamarin.Android, but not abandoning .netX.0-android / .netX.0-iOS? Or do you plan to remove those as well?

**Comment by pauldendulk** [comment, marked as answer]
I think we should still support the Xamarin.Android and Xamarin.iOS packages in v5 and I will update the roadmap to make that clear.

Actually, I did not have a clear picture of the status of the underlying libraries. I have been looking into this just now.  This is what MAUI is built on. With UI frameworks quickly coming and going, the support for .NET Android/iOS might even last longer than for MAUI itself.

**Reply by MateuszLas421** [reply]
Thank you for your reply and development Mapsui.
